app/validation/validator.py

```diff
@@ -1434,6 +1457,19 @@ def _get_list_names(json_to_validate):
                         list_names.append(block['for_list'])
         return list_names
 
+    @staticmethod
+    def _get_block_ids(json_to_validate):
+        block_ids = []
+        for section in json_to_validate['sections']:
+            for group in section['groups']:
+                for block in group['blocks']:
+                    block_ids.append(block['id'])
+                    for sub_block in {'add_block', 'edit_block', 'remove_block'}:
```

Comment:
This should also include `add_or_edit_block`

Reply:
Ah ofc. Added.